main.go

```diff
@@ -159,53 +163,61 @@ func before(c *cli.Context) error {
 
 func start(c *cli.Context) error {
 	names := c.Args()
+        if c.GlobalBool("oneshot") {
+		log.SetLevel(log.DebugLevel)
+		log.Info("OneShot run: " + time.Now().String())
+	        notifier.StartNotification()
+		if err := actions.Update(client, names, cleanup, noRestart); err != nil {
+			log.Println(err)
+		}
+		notifier.SendNotification()
```

Comment:
It would be nice to put line 169-173 into a method, so we could also call that method on line 189 instead of having duplicated code there.